overview.md

```diff
@@ -43,130 +43,173 @@ steps:
 
 ```YAML
 pool:
   vmImage: ubuntu-latest
 steps:
-  - task: InstallMATLAB@1
-    inputs:
-      release: R2023b
-  - task: RunMATLABCommand@1
-    inputs:
-      command: myscript
+- task: InstallMATLAB@1
+- task: RunMATLABTests@1
+  env:
+    MLM_LICENSE_TOKEN: $(myToken)
+```
+
+### Build Across Multiple Platforms
+The **Install MATLAB** task supports the Linux, Windows, and macOS platforms. Use a `matrix` job strategy to run a build using the MATLAB build tool on all the supported platforms. This pipeline runs three jobs.
+
+```YAML
+strategy:
+  matrix:
+    linux:
+      imageName: ubuntu-latest
+    windows:
+      imageName: windows-latest
+    mac:
+      imageName: macOS-latest
+pool:
+  vmImage: $(imageName)
+steps:
+- task: InstallMATLAB@1
+- task: RunMATLABBuild@1
+  inputs:
+    tasks: test
 ```
 
 ## Tasks
 You can access the extension tasks and add them to your pipeline when you edit your pipeline in Azure DevOps. 
 
-![tasks](https://user-images.githubusercontent.com/48831250/193909715-1c90eb94-d89d-458d-80bc-2fee4c20c760.png)
+![tasks](https://github.com/mathworks/matlab-azure-devops-extension/assets/48831250/d48ddb8b-a87f-4334-a301-64293b822647)
 
 ### Install MATLAB
-Install a specific version of MATLAB. Specify the task in your pipeline YAML using the `InstallMATLAB` key.
+Use the **Install MATLAB** task to run MATLAB code and Simulink models on a Microsoft-hosted agent. When you specify this task as part of your pipeline, the task installs your preferred MATLAB release (R2021a or later) on a Linux&reg;, Windows&reg;, or macOS agent and prepends it to the `PATH` system environment variable. If you do not specify a release, the task installs the latest release of MATLAB.
+
+Specify the **Install MATLAB** task in your YAML pipeline as `InstallMATLAB@1`. The task accepts optional arguments.
+
+| Argument  | Description |
+|-----------|-------------|
+| `release` | <p>(Optional) MATLAB release to install. You can specify R2021a or a later release. By default, the value of `release` is `latest`, which corresponds to the latest release of MATLAB.<p/><p>**Example**: `release: R2023b`<br/>**Example**: `release: latest`</p>
+| `products` | <p>(Optional) Products to install in addition to MATLAB, specified as a list of product names separated by spaces. You can specify `products` to install most MathWorks products and support packages. For example, `products: Deep_Learning_Toolbox` installs Deep Learning Toolbox&trade; in addition to MATLAB.</p><p>The task uses [MATLAB Package Manager](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md) (`mpm`) to install products. For a list of supported products and their correctly formatted names, see [Product Installation Options](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md#product-installation-options).</p><p>**Example**: `products: Simulink`</br>**Example:** `products: Simulink Deep_Learning_Toolbox`</p>
```

Comment:
Can we show an example that lists the products on their own line? This is a bit of YAML-Fu that many customers might trip over (and it is different on Azure DevOps than GitHub Actions for example. Perhaps something like:

    products: >
       Simulink
       Deep_Learning_Toolbox

Reply:
Good point, Andy. I can include this example:

pool:
  vmImage: ubuntu-latest
steps:
  - task: InstallMATLAB@1
    inputs:
      products: Parallel_Computing_Toolbox
  - task: RunMATLABTests@1
    inputs:
      useParallel: true

Reply:
This example doesn't show the list feature though. Can we add an additional product?

Reply:
I don't think that would be necessary. We say that they should separate products with a space and show an inline example: 
products: Simulink Deep_Learning_Toolbox

Reply:
@davidbuzinski: Can you come up with something that addresses Andy and Mark's comment? I can't think of a meaningful example.

Reply:
Updated the example, @mcafaro. Even though I could run the pipeline, can you make sure the indentation is as expected?